packages/design-system/src/scripts/components/DateField/DateField.jsx

```diff
@@ -166,6 +169,10 @@ DateField.defaultProps = {
 };
 
 DateField.propTypes = {
+  /**
+   * Adds `autocomplete` attributes `bday-day`, `bday-month` and `bday-year` to the corresponding `<DateField>` inputs
+   */
+  autoComplete: PropTypes.bool,
```

Comment:
@nichia I was looking over this and I realized we probably should have used the name `birthdayAutocomplete` instead of `autocomplete`....